models/StudentsHomework.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace app\models;
+
+use Yii;
+
+/**
+ * This is the model class for table "students_homework".
+ *
+ * @property integer $students_id
+ * @property integer $homework_id
+ *
+ * @property Homework $homework
+ * @property Students $students
+ */
+class StudentsHomework extends \yii\db\ActiveRecord
```

Comment:
If this model is only for creating many-to-many relations - you can not create it.